test/functional/f5_plugins/test_active_standby.py

```diff
@@ -42,15 +42,11 @@ def test_active_standby(HeatStack, symbols, F5PluginTemplateLoc):
             'cluster_name': symbols.cluster_name
         }
     )
+    a = ManagementRoot(symbols.bigip1_ip, symbols.bigip1_un, symbols.bigip1_pw)
```

Comment:
Paul, could you change the name of a and b objects to reflect bigip1 and bigip2?